tests/integration/targets/docker_plugin/tasks/main.yaml

```
@@ -0,0 +1,24 @@
+- name: Create random name prefix
+  set_fact:
+    name_prefix: "cvmfs/overlay2-graphdriver"
+    plugin_names: []
+
+- debug:
+    msg: "Using name prefix {{ name_prefix }}"
+
+- block:
+  - include_tasks: run-test.yml
+    with_fileglob:
+    - "tests/*.yml"
+
+  always:
+  - name: "Make sure plugin is removed"
+    docker_plugin:
+      plugin_name: "{{ item }}"
+      state: absent
```

```suggestion
      state: absent
      force_remove: true
```
To make sure they are really gone (even if still enabled).